src/components/ContextMenuView/ChatContextMenuView.tsx

```diff
@@ -8,6 +8,7 @@ import {Alert, NativeSyntheticEvent} from 'react-native';
 import ChatSchema from '../../database/schema/ChatSchema';
 import ShareUtils from '../../utils/share';
 import useMessageHook from '../../hooks/core/chat/useMessageHook';
+import {Anonimity} from '../../../types/component/AnonymousChat/BaseChannelItem.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Anonimity` type is a good addition to ensure the correct usage of chat types in the context menu. This will help maintain consistency and avoid potential bugs due to incorrect or inconsistent usage of chat types.
```diff
+ import {Anonimity} from '../../../types/component/AnonymousChat/BaseChannelItem.types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->